daml/Marketplace/Distribution/Syndication/Investor.daml

```diff
@@ -16,15 +17,6 @@ template Role
 
     controller investor can
 
-      nonconsuming SelectSettlementAccounts : ContractId Custody.SettlementInfo
```

Comment:
Is this removed because we will simply use the logic in Hierarchical.getAccount to find the settlement account that we need when creating instruction? (so basically, we don't give the user the right to choose which account they would like to use)

Reply:
Ah I see now, thanks!